Casks/t/tpvirtual.rb

```diff
@@ -0,0 +1,25 @@
+cask "tpvirtual" do
+  version "3"
+  sha256 "ed4126989c26c2d9b0e2b48ec454340eb3f50042797a5b4a201435d3cdf80e1f"
+
+  url "https://virtual.trainingpeaks.com/TPVirtual-Installer_v#{version}.dmg"
+  name "TrainingPeaks Virtual"
+  desc "Indoor cycling game"
+  homepage "https://www.trainingpeaks.com/virtual/"
+
+  livecheck do
+    url :homepage
+    strategy :page_match do |page|
+      page[/TPVirtual-Installer_v(\d+)\.dmg/i, 1]
+    end
```

Comment:
```suggestion
    regex(/href=.*?TPVirtual[._-]Installer[._-]v?(\d+(?:\.\d+)*)\.dmg/i)
```

The `PageMatch` strategy is used by default since no specific strategy applies to the URL.